test/examples/ucp_client_server.c

```diff
@@ -589,7 +589,7 @@ int main(int argc, char **argv)
         }
     } else {
         /* Client side */
-        status = start_client(ucp_worker, server_addr, &ep);
+        status = (ucs_status_t)start_client(ucp_worker, server_addr, &ep);
```

Comment:
start_server/start_client should return ucs_status_t

Reply:
fixed